modules/aidebug/cpp/aidebug.cpp

```diff
@@ -0,0 +1,129 @@
+﻿#include "../src/common/mmo.h"
+#include "../src/map/ai/ai_container.h"
+#include "../src/map/entities/mobentity.h"
+#include "../src/map/entities/npcentity.h"
+#include "../src/map/mob_modifier.h"
+#include "../src/map/utils/mobutils.h"
+#include "../src/map/utils/moduleutils.h"
+#include "../src/map/zone.h"
+#include "../src/map/zone_entities.h"
+
+#include <vector>
+
+class AIDebugModule : public CPPModule
```

Comment:
I think this could be named better, this isn't for debugging AI, this is for debugging pathfinding

Reply:
As pathfinding is a subset of AI and this module is to be extended to power further AI debugging features such as custom packet handling for sending entity state and different behavior data back to the client, that's why I've gone with the name AIDebugModule